boards/rp-pico/examples/pico_uart_irq_buffer.rs

```diff
@@ -19,7 +19,8 @@
 
 // These are the traits we need from Embedded HAL to treat our hardware
 // objects as generic embedded devices.
-use embedded_hal::{digital::v2::OutputPin, serial::Write as UartWrite};
+use embedded_hal::digital::OutputPin;
+use embedded_hal_0_2::serial::Write as UartWrite;
```

Comment:
I think the updated alternative for `embedded_hal_0_2::serial::Write` would be [`embedded_hal_nb::serial::Write`](https://docs.rs/embedded-hal-nb/latest/embedded_hal_nb/serial/trait.Write.html#)

Reply:
I have pushed changes to replace usage of `embedded_hal_0_2::serial::Write` with `embedded_hal_nb::serial::Write`